contracts/seed/Seed.sol

```diff
@@ -559,8 +569,8 @@ contract Seed {
                 "Seed: should transfer seed tokens to refund receiver"
             );
         } else {
-            uint256 seedAmountRequired = (hardCap * PRECISION) / price;
-            uint256 feeAmountRequired = (seedAmountRequired * fee) / PRECISION;
+            uint256 seedAmountRequired = (hardCap * PRECISION) / classes[0].price;
```

Comment:
Why does those changes appear randomly in already 3d PR?)

Reply:
removed